src/app/features/dashboard-widgets/entity-count-dashboard-widget/entity-count-dashboard/entity-count-dashboard.component.html

```diff
@@ -67,7 +67,20 @@
       <ng-container matColumnDef="label">
         <td *matCellDef="let group">
           @if (!group.groupedByEntity) {
-            <span>{{ group.label }}</span>
+            @if (group.label === undefined && group.isInvalidOption) {
```

Comment:
why is label undefined important here? isn't isInvalidOption enough as condition?